webapp/app/js/perf.js

```diff
@@ -335,10 +344,20 @@ perf.controller('PerfCtrl', [ '$state', '$stateParams', '$scope', '$rootScope',
 perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
                                             projects, optionCollectionMap,
                                             timeRange, thServiceDomain,
-                                            seriesSummary) {
+                                            seriesSummary, defaultProjectName,
+                                            defaultPlatform) {
   $scope.timeRange = timeRange;
   $scope.projects = projects;
-  $scope.selectedProject = projects[0];
+  if (defaultProjectName) {
+    for (var i in projects) {
```

Comment:
This is all good.  But you *could* use ``_.findWhere`` if you wanted:  http://underscorejs.org/#findWhere